config/locales/idv/fr.yml

```diff
@@ -240,9 +240,9 @@ fr:
           code à usage unique. Ceci est pour aider à vérifier votre identité.
         failed_number:
           alert_text: Nous n’avons pas pu vous associer à ce numéro.
-          gpo_alert_html: Si vous n’avez pas <strong>d’autre</strong> numéro de téléphone
-            à essayer, %{link_html}.
+          gpo_alert_html: Essayez un <strong>autre</strong> numéro ou %{link_html}.
           gpo_verify_link: vérifiez plutôt par courrier
```

Comment:
The other translations with this tag just say "verify by mail". Do we want to update this one to match? It says "verify by mail instead".
```suggestion
          gpo_verify_link: vérifiez par courrier
```

Reply:
@artfulaction - thoughts? 🤔

Reply:
Let's remove "instead" from all (including English) to keep it consistent. Thanks for catching that!

Reply:
instead removed:

<img width="633" alt="LG-10511-fr-instead" src="https://github.com/18F/identity-idp/assets/1261794/34880fd5-aa21-4c10-9bbf-6ff222187829">
<img width="629" alt="LG-10511-es-instead" src="https://github.com/18F/identity-idp/assets/1261794/00f51dfa-cd53-4308-9fab-97f4c56d3a0a">
<img width="641" alt="LG-10511-en-instead" src="https://github.com/18F/identity-idp/assets/1261794/f67fd27a-600c-405d-bc59-a504887a54d8">

Reply:
Looks good to me!